lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1415,7 +1222,7 @@ export class UserAgentApplication {
    * @param hash
    */
   private deserializeHash(hash: string) {
-    hash = this.getHash(hash);
+    hash = Utils.getHashFromUrl(hash);
```

Comment:
Reassigning parameter values can be dangerous, I would like to see this as a new variable (or even just passed in directly into `Utils.deserialie`.

Reply:
Changed the name of the function argument